### ISSUE TITLE
Feature requests: Ability to ban anyone with a flag next to their username 

### ISSUE THREAD

**Issue by booblikum** [satsukitv/twitter-block-chain#92]
Is it possible to ban anyone with a hashtag for example like `NAFO` here? or a country's flag next to their name? 
They clutter up the feed and keep spamming. 

<img width="238" alt="image" src="https://user-images.githubusercontent.com/47491297/192743912-97f120aa-3205-4067-8c6a-1d50d74a61fc.png">

**Comment by Jka4** [comment]
but ban everyone with rusland flag be much better and easier 🥰

**Comment by BigDvsRL** [comment]
Yeah, would be cool to add some Filters. Maybe the Block is just Triggered if specific Keywords in name or bio. 
And maybe a Function to mass Protect Channels...
